spec/operators/take-spec.ts

```diff
@@ -1,110 +1,139 @@
 import { expect } from 'chai';
-import { hot, cold, expectObservable, expectSubscriptions } from '../helpers/marble-testing';
 import { take, mergeMap } from 'rxjs/operators';
 import { range, ArgumentOutOfRangeError, of, Observable, Subject } from 'rxjs';
+import { TestScheduler } from 'rxjs/testing';
+import { observableMatcher } from '../helpers/observableMatcher';
 
 declare function asDiagram(arg: string): Function;
 
 /** @test {take} */
 describe('take operator', () => {
+  let testScheduler: TestScheduler;
+
+  beforeEach(() => {
+    testScheduler = new TestScheduler(observableMatcher);
+  });
+
   asDiagram('take(2)')('should take two values of an observable with many values', () => {
-    const e1 =  cold('--a-----b----c---d--|');
-    const e1subs =   '^       !            ';
-    const expected = '--a-----(b|)         ';
+    testScheduler.run(({ cold, expectObservable, expectSubscriptions }) => {
+      const e1 =  cold('--a-----b----c---d--|');
+      const e1subs =   '^-------!------------';
+      const expected = '--a-----(b|)         ';
```

Comment:
We'd like to be able to use Prettier with the codebase and that means the alignment will need to happen within the marble strings, like this:

```
const e1 = cold('  --a-----b----c---d--|');
const e1subs = '   ^-------!------------';
const expected = ' --a-----(b|)         ';
```

As Prettier will collapse multiple adjacent spaces outside of the string literals.

Reply:
ok! I'll do that right now